vendor/github.com/onsi/gomega/go.mod

```diff
@@ -1,17 +0,0 @@
-module github.com/onsi/gomega
-
-require (
```

Comment:
why was this removed?

Reply:
Thanks for pointing it out. It got removed accidently.